src/overlays/actors/ovl_Obj_Jg_Gakki/z_obj_jg_gakki.h

```diff
@@ -7,7 +7,8 @@ struct ObjJgGakki;
 
 typedef struct ObjJgGakki {
     /* 0x000 */ Actor actor;
-    /* 0x144 */ char unk_144[0x90];
+    /* 0x144 */ SkelAnime skelAnime;
+    /* 0x188 */ ColliderCylinder colliderCylinder;
```

Comment:
```suggestion
    /* 0x188 */ ColliderCylinder collider;
```
When there is no additional collider, we normally just call it collider.

Reply:
I'll add that to my notes, thanks for the feedback!
(I'm adding the suggestion through github, if it makes Jenkins mad I'll fix it up manually tonight)